keps/sig-multicluster/4322-cluster-inventory/README.md

```diff
@@ -379,7 +382,28 @@ command or function to work in the same way in another tool. This can
 further enhance the usability and adoption of different cluster
 manager.
 
-### Notes/Constraints/Caveats (Optional)
+### Notes/Constraints/Caveats
+
+#### How to Utilize the API
+This section provides guidance on the recommended way to interact with the API. It is suggested that
+all clusterProfile objects, which represent a single cluster in the inventory, should be emitted
+on a dedicated management/hub Kubernetes cluster. This approach allows consumers to have a single
+integration point to access all the information within a cluster inventory. A controller can be run 
+on the management or hub cluster to offer high-level functionalities over this inventory of clusters.
+
+#### Approaches for Structuring Clusters within an Inventory
```

Comment:
I'm not sure I understand how this section is worded. The namespace is to make explicit groups of ClusterProfiles and allow multiple groups.

It is a requirement at this point I think: "Allow multiple disjoints groups of clusters".
And the only way to support that in K8s is via a Namespace.

I think it's not a recommendation, it's a mandate.

Reply:
This is within an inventory.  The recommended way is to place them in the same namespace. Do you want to mandate that?